src/hooks/safe-apps/useAppsSearch.ts

```diff
@@ -0,0 +1,25 @@
+import { useMemo } from 'react'
+import Fuse from 'fuse.js'
+import { SafeAppData } from '@gnosis.pm/safe-react-gateway-sdk'
+
+const useAppsSearch = (apps: SafeAppData[], query: string): SafeAppData[] => {
+  const fuse = useMemo(
+    () =>
+      new Fuse(apps, {
```

Comment:
I saw a [tweet](https://twitter.com/diegohaz/status/1561036326849372163) about a lesser-known fuzzy search package get some traction the other day. Maybe it's worth checking out.

Reply:
I saw this tweet, too. While it's no doubt faster, we probably won't notice anything because our set is relatively small (less than 100 apps). Also, the lib hasn't been updated since May 18, while fuse.js has sponsorships by companies (sentry, etc) and has somewhat brighter future when it comes to future maintenance